az-digital/az_quickstart
# Closes #1396: Added 3-col view display for AZ Person

## Description
The default view display is 4 columns on large and x-large, so I added a 3-column option. Also changed the default medium layout to 2-column.

## Related issues




Closes #1396
Related #1645

## How to test
Run on local or Probo, create new page, add view, select AZ Person > Grid - 3 Col display, confirm the layout is three columns instead of four. Install demo content if needed.

## Types of changes
Added one additional view display that was cloned from Grid Page, using Duplicate as Block. Then adjusted the Format > Bootstrap Grid settings to specify the column prefs. Grid Page at 4-cols in still the default.

### Arizona Quickstart (install profile, custom modules, custom theme)
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal core
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal contrib projects
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

## Checklist
